src/test/java/com/github/jnidzwetzki/bitfinex/v2/test/IntegrationTest.java

```diff
@@ -320,6 +302,7 @@ public void testTickerStream() {
 			Assert.assertTrue(bitfinexClient.isTickerActive(symbol));
 
 			orderbookManager.unsubscribeTicker(symbol);
```

Comment:
Maybe we should return a Future<Boolean> (in a later refactoring) which returns true as soon as these operations (e.g. unsubscribe and subscribe) are performed.

Reply:
do you mean Future from orderbookManager? Sure - it's not a bad idea, whatever benefits the user and ease the development